index.html

```diff
@@ -9,44 +9,216 @@
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <img src="project-img.png" alt="project">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
+        </div>
+      </div>
+
+      <!-- card -->
+      <div class="card">
+        <div class="card-content">
+          <h2 class="card-title">Multi-Post Stories Gain+Glory</h2>
+          <ul class="skills">
+            <li class="skill">Ruby on Rails</li>
+            <li class="skill">Css</li>
+            <li class="skill">html</li>
+            <li class="skill">Javascript</li>
+          </ul>
+          <button class="btn btn-skill">See Project</button>
```

Comment:
your button are well styled they look exactly like the design but still missing the hover and press effects 

![image (20)](https://github.com/RolandAmini/portfolio-mobile/assets/59707859/13e99ca3-a98d-4d91-b56c-798d1f46ff8e)